packages/financial-templates-lib/src/helpers/math.js

```diff
@@ -31,7 +33,26 @@ const isDeviationOutsideErrorMargin = (observedValueBN, expectedValueBN, scaling
   return deviationBN.abs().gt(errorMarginBN);
 };
 
+const formatPriceToPricefeedPrecision = (price, priceFeedDecimals, identifier) => {
```

Comment:
this makes a lot of sense as a method and is something we should have actually done a while ago. Previously we had a few UMIP implementations that we were actually not conforming to as a result of not having something like this, such as within the liquidator/disputer. It only really makes a difference on the margin though (or in the case of the funding rate related bots!).